Framework/Subset_of_CorLib/System/DateTime.cs

```diff
@@ -93,13 +88,18 @@ public struct DateTime
         private const ulong TickMask = 0x7FFFFFFFFFFFFFFFL;
         private const ulong UTCMask = 0x8000000000000000L;
 
-        public static readonly DateTime MinValue = new DateTime(MinTicks);
-        public static readonly DateTime MaxValue = new DateTime(MaxTicks);
+        // Ticks at 12:00 AM January 1, year 1601 A.D.

```

Comment:
It's worth putting a more detailed comment here explain why this is done this way. (e.g. shifting to compat with desktop without breaking existing HAL). We'll also need to ensure we highlight this behavioral change in our release notes for this version.